core/system/src/mpu/vmpu.c

```diff
@@ -165,6 +165,41 @@ static void vmpu_sanity_check_box_namespace(int box_id, const char *const box_na
     } while (box_namespace[length]);
 }
 
+static void vmpu_box_index_init(uint8_t box_id, uint32_t context_size, uint32_t heap_size)
+{
+    if (box_id == 0)
+    {
+        /* box 0 still uses the main heap to be backwards compatible */
+        g_svc_cx_context_ptr[0] = __uvisor_config.heap_start;
+        heap_size = ((void*)__uvisor_config.heap_end -
```

Comment:
@niklas-arm please add security check to verify that __uvisor_config.heap_start/__uvisor_config.heap_end are inside SRAM and outside of uVisor.